js/weather.js

```diff
@@ -1,35 +1,91 @@
 $(function () {
+  let selectedBox = 'current';
+  let apiData = {};
+
+  $('.toggle-buttons').click(function (e) {
+    // prevent checked box to be unchecked on click
+    if (!$(this).is(':checked')) {
+      e.preventDefault;
+      return false;
+    }
+    $(this).siblings().attr('checked', false); // uncheck checked siblings
+    selectedBox = $(this)[0].value;
+
+    if (Object.keys(apiData).length) {
+      displayResults();
+    }
+  });
+
   // React to hitting enter in the text box instead of clicking submit.
-  $("#search-weather").submit(function(e) {
+  $('#search-weather').submit(function (e) {
     e.preventDefault();
     clearResult('');
 
     // Get the weather info for the selected search location.
-    $.get("/weather.php?query=" + $("#search").val(), function (data) {
+    $.get('/weather.php?query=' + $('#search').val(), function (data) {
+      apiData = data;
+      console.log(data);
```

Comment:
I assume this console.log was just for testing. Be sure these are removed for PR.